.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "eslint:recommended",
+    "overrides": [
```

Comment:
The overrides block is a place to put custom linting instructions that you might need for testing. We expect you to have two objects in this array, one for Jest and one for Cypress files. This configuration data is provided in the lessons on unit and e2e testing respectively.